app.js

```diff
@@ -1,16 +1,17 @@
 var debug = require('debug')('paysage:app');
 
+var World = require('./World');
+
 var express = require('express');
 var path = require('path');
 var logger = require('morgan');
 var cookieParser = require('cookie-parser');
 var bodyParser = require('body-parser');
 var exphbs = require('express-handlebars');
 
-module.exports = function(someCodeObjects) {
-
-  var codeObjects = someCodeObjects || {};
+module.exports = function(world) {
   var app = express();
+  var world = world ? world : new World();
```

Comment:
`var` here is redundant and misleading. The `world` variable already exists in this scope by virtue of being a formal parameter.

Just drop the `var`.

Reply:
Or rename the `world` parameter to `maybeWorld`?